packages/fx-core/src/common/deps-checker/checkerFactory.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Adjust code owner for deps-checker, as well as tests.

Reply:
Added